### PR TITLE
[on hold] adds a true baking recipe about EUbOPEN deposition

### PR DESCRIPTION
**Pull request by robertgiessmann**
EFPIA feedback was that the cookbook felt like a strange collection of papers, and that the "cookbook style" is not evident. 

This recipe is a first attempt into that direction, writing a recipe more like a traditional-cooking-and-baking-cookbook. (and assuming you cooked before, and/or have read the background on this class of dishes and the basic cooking introduction)

What do you think? Pinging somewhat randomly @daniwelter @proccaserra @nsjuty and not assigning anyone.

### CODE REVIEW

File: content/recipes/applied-examples/eubopen.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+(fcb-eubopen)=
+# FAIRification of High-Content Screening data using Bioimage Archive
+
+````{panels_fairplus}
+:identifier_text: TODO
+:identifier_link: TODO
+:difficulty_level: 3
+:recipe_type: applied_example
+:reading_time_minutes: 15
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:9]: needs to be longer to be 15 mins...

File: content/recipes/applied-examples/eubopen.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Abstract
+
+You run high-content screens to test the effect of compounds on cells. You want to make your data FAIR. This recipe is for you. 
+
+
+## Recipe
+
+1. Download the Python scripts and the metadata template Excel sheet from [https://doi.org/10.5281/zenodo.6325622](https://doi.org/10.5281/zenodo.6325622).
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:23]: verified files exist in that archive

File: content/recipes/applied-examples/eubopen.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+2. Organize your metadata according to the metadata template sheet.
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:24]: i would state what the template is based on if necessary, or add a sentence in brackets or such)

File: content/recipes/applied-examples/eubopen.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+3. Generate filelists with the Python scripts.
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:25]: there are a few py scripts .. worth stating which one

File: content/recipes/applied-examples/eubopen.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+4. Zip the raw data, and upload the zip files as well as the individual processed images to [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/).
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:26]: i would use the submit link here: https://www.ebi.ac.uk/bioimage-archive/submit/

File: content/recipes/applied-examples/eubopen.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+5. Run the automated analysis of your data with the Python scripts.
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:27]: state which or order?

File: content/recipes/applied-examples/eubopen.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+6. Upload your analysis data and the aggregated version to Zenodo.
```
Comment (nsjuty) [inline content/recipes/applied-examples/eubopen.md:28]: link to recipe to submit to zenodo?

**Review by nsjuty — CHANGES_REQUESTED** [review]
i suggest a few additions..

File: content/recipes/applied-examples/eubopen.md (hunk 3 above)
Comment (proccaserra) [reply content/recipes/applied-examples/eubopen.md:24]: describe how this template was created ? was it supplied to you by Bioimage? was it custom made ? Is the template identified, versioned etc..?

File: content/recipes/applied-examples/eubopen.md (hunk 4 above)
Comment (proccaserra) [reply content/recipes/applied-examples/eubopen.md:25]: +1

File: content/recipes/applied-examples/eubopen.md (hunk 2 above)
Comment (proccaserra) [reply content/recipes/applied-examples/eubopen.md:23]: add link to github repo:
https://github.com/rgiessmann/data-management-for-high-content-screening

File: content/recipes/applied-examples/eubopen.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+
```
Comment (proccaserra) [inline content/recipes/applied-examples/eubopen.md:30]:
```suggestion

````{dropdown} **References**
{footbibliography}
```
Ideally, all recipes should have a couple of bibliographic references

File: content/recipes/applied-examples/eubopen.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
+(fcb-eubopen)=
+# FAIRification of High-Content Screening data using Bioimage Archive
+
+````{panels_fairplus}
+:identifier_text: TODO
+:identifier_link: TODO
+:difficulty_level: 3
+:recipe_type: applied_example
```
Comment (proccaserra) [inline content/recipes/applied-examples/eubopen.md:8]:
```suggestion
:maturity_level: 1
:maturity_indicator: 1
```

NOTE: the values for the maturity level and indicator are examples. would need to be added.

**Comment by proccaserra** [comment]
@robertgiessmann thx for this. The layout of the current existing recipes follow this (especially when a jupyter notebook is present). 
What you are hinting at is being as dry/compact / directive as possible. 
This works for some audience or some type of recipes but sometimes, in more complex cases, the audience looks for 
 wealth of details and explanations and this may vary depending of the level of expertise of the audience. I have received requests for instance to expand on some aspects of the Ontology building robot recipe.

**Comment by proccaserra** [comment]
@robertgiessmann can you specify the order in which the scripts need to be run. 
There are 7 scripts in your repo and the reader would need further guidance.
One more question: the excel template: Is it one you specifically produced for Eubopen or is it the standard template used by BioImage Archive?
Many thx!
tchuss

**Comment by robertgiessmann** [comment]
hi @proccaserra @nsjuty  , indeed, I was aiming for something very compact here. I looked into my actual baking books, and found recipes with 3 to 9 steps, with a median of 6 steps (roughly ;). 

Most details I found to be irrelevant if you are really providing a baking recipe, like "how the template was created" -- the user who applies this recipe does not need to know it necessarily. Indeed some might be interested in the background, but that's not the responsibility of a baking recipe. 

Regarding the scripts and their order: there is a publication upcoming by Amelie, in which more detailled steps are given. Looking at 6 steps and an obviously quite proficient user here, I assume that the order of the scripts is figured out quickly by the user, and is not a real necessity. But let's wait for the manuscript to be published, then we can point from the recipe to there.

Regarding the question: this is a custom-created template for EUbOPEN <-> BioImage Archive; there are/were no actual templates around for this datatype.

**Comment by proccaserra** [comment]
@robertgiessmann 

To follow-up on the cooking metaphor, the order of some steps are important and it should be clear. It is always a bit of a let down when reading a recipe and having to open another book to get the key information. We can avoid this UX hurdle by citing the manuscript (preprint DOI, which can be updated when the actual publication is out).

Regarding the template, is it now recommended or used by BioImage Archive?
In other words, did they adopt it? 

Can you add any constraint (if any!) regarding the template (e.g. instrument independence? is it possible to provide ontology based annotations etc..etc..) ?

**Comment by robertgiessmann** [comment]
> Regarding the template, is it now recommended or used by BioImage Archive?
> In other words, did they adopt it?

Not as far as I know.

> Can you add any constraint (if any!) regarding the template (e.g. instrument independence? is it possible to provide ontology based annotations etc..etc..) ?

All possible and fully independent, however the output is disappointing from Linked-Open-Data point of view -- it's just a bunch of lines in a BioImage Archive file; see e.g. https://www.ebi.ac.uk/biostudies/studies/S-BIAD145 , and e.g. https://www.ebi.ac.uk/biostudies/files/S-BIAD145/filelist_raw_data_cq1.json

**Comment by proccaserra** [comment]
closing since recipe has been released
